src/associations/one-to-many.js

```diff
@@ -152,14 +152,14 @@ export const scopeScope = (server, a, b, names) => {
   });
 };
 
-export const destroy = (server, a, b, names) => {
+export const destroy = async (server, a, b, names) => {
   server.route({
     method: 'DELETE',
-    path: `${prefix}/${names.a.singular}/{aid}/${names.b.plural}`,
+    path: `${prefix}${names.a.singular}/{aid}/${names.b.plural}`,
```

Comment:
this things was formatted like `\`${prefix}${names.a.singular}/{aid}/${names.b.plural}\``